docsrc/commands.rst

```diff
@@ -140,3 +140,45 @@ This can be changed using ``require_command`` property.
    local parser = argparse()
       :require_command(false)
    parser:command "install"
+
+Command summaries
```

Comment:
Capital 's'?

Reply:
Other heading in the docs aren't capitalized